index.html

```diff
@@ -112,27 +114,31 @@ <h5 class="cardtitle fw-bolder mx-2 text-center opacity-100">Network</h5>
       </div>
 
       <div class="card register-btn h-50 p-2 mx-auto d-flex flex-row justify-content-evenly align-items-center
-                rounded-0 mb-3">
-        <a class="cardtitle fw-bold py-3 text-center mx-3 opacity-100 text-light text-decoration-none
+                rounded-0  my-4 ">
+        <a class="cardtitle fw-bold py-3 text-center mx-3 opacity-100 text-light
```

Comment:
- [ ] [OPTIONAL] In the `main program section` , I can see that there is still a bit of misalignment. As you are already using the flex box I suggest you use `flex-basis` property on the `title`. The flex-basis `CSS` property sets the initial main size of a flex item. It sets the size of the content box unless otherwise set with `box-sizing`.  This way even when the sizes of different `titles` is different the initially designated size to the `element` will make the rest of the main cards look like they are aligned. 

Your design: 

<img width="376" alt="Screenshot 2023-03-16 at 11 24 19 AM" src="https://user-images.githubusercontent.com/44798044/225540634-3a3a07a4-e7c0-448d-b840-7823860685d6.png">

<img width="1044" alt="Screenshot 2023-03-16 at 11 25 22 AM" src="https://user-images.githubusercontent.com/44798044/225540650-2b63338d-d0c2-4d08-a752-014d1f15b6bc.png">

Behance design:

<img width="644" alt="Screenshot 2023-03-16 at 11 33 28 AM" src="https://user-images.githubusercontent.com/44798044/225540710-37790ac0-9bfd-4f6b-88e1-cbae0f7c168f.png">